File: Assignment 1/C# version/SumOfThree.cs
using System;

class SumOfThree
{
	public static void Main(string[] args)
	{
		Console.Write("Provide a three digit number:  ");
		int input = Convert.ToInt32(Console.ReadLine());
		int sum = 0;
		int sum = input % 10;
		input /= 10;
		sum += input % 10;
		sum += input / 10;
		Console.WriteLine("Sum of digits: " + sum);
	}
}

File: Assignment 1/Java version/SumOfThree.java
import java.util.Scanner;

public class SumOfThree {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		System.out.print("Provide a three digit number: ");
		int input = scan.nextInt();
		scan.close();
		int sum = input % 10;
		input /= 10;
		sum += input % 10;
		sum += input / 10;
		System.out.println("Sum of digits: " + sum);
	}
}
